BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -375,18 +375,18 @@ public final class BlueprintView: UIView {
         }
 
         Logger.logViewUpdateEnd(view: self)
-        let viewUpdateEndDate = Date()
+        let viewUpdateEndTime = CACurrentMediaTime()
```

Comment:
Is this in seconds still? Fine if it's not, just curious.

Actually, why don't I just not be lazy and check, yep:

> Returns the current absolute time, in seconds.